templates/partials/topbar.html.njk

```diff
@@ -11,7 +11,6 @@
   <div class="flex items-center">
     <nav id="topbar-nav" class="topbar-nav items-center">
       <a href="/speakers/" class="link--promiselinen link--nav">Speakers</a>
-      <a href="{{ contents['scholarships.json'].url }}" class="link--promiselinen link--nav">Scholarships</a>
```

Comment:
Are we blocking any further planned pages on the main nav so that we need to move scholarships to the footer? If not, I am hesitating to put it to the footer since scholarship is an essential part of the confs. Can we ask more people here about their opinion?

Reply:
The right long-term place is the info page linked from the top nav I think
https://github.com/jsconf/festival-x.jsconf.eu/blob/master/templates/pages/info.html.njk#L16